### PR TITLE
Update registry.json

### PR DESCRIPTION
**Pull request by ThePeer**
added FLOW

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+       "poolId": "58920efd533bc451664937d03c14176b0693a1800326933f3abbc6e8",  
+       "social": {              
+         "website":"https://theriver906928134.wordpress.com", 
+         "twitter": "the_FLOW_pool",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "the_FLOW_pool",                                             
+       "operatorName": "FLOW pool",
```
Comment (leo42) [inline registry.json:13]: Please use a real name or leave empty.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (ThePeer) [reply registry.json]: replaced "FLOW pool" with "Peer"

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+       "social": {              
+         "website":"https://theriver906928134.wordpress.com", 
+         "twitter": "the_FLOW_pool",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "the_FLOW_pool",                                             
+       "operatorName": "Peer",                                  
+
```
Comment (Bez625) [inline registry.json:15]: You seem to be missing a brace:

```suggestion
},
```

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json [hunk 3]
```diff
@@ -1,4 +1,18 @@
...
+       "poolId": "58920efd533bc451664937d03c14176b0693a1800326933f3abbc6e8",  
+       "social": {              
+         "website":"https://theriver906928134.wordpress.com", 
+         "twitter": "the_FLOW_pool",
+         "facebook": " ",  
+         "instagram": " ",
+         "youtube": " "
+       },
+       "telegram": "the_FLOW_pool",                                             
+       "operatorName": "Peer",
```
Comment (Bez625) [inline registry.json:14]:
```suggestion
       "github": ""                               
```

Comment (Bez625) [inline registry.json:14]:
```suggestion
       "operatorName": "Peer",  
       "github": ""                                    
```

**Pull request merged**